README.md

```diff
@@ -38,6 +42,16 @@ bootstraps
 masters
 agents
 agents_public
+
+[linux:children]
```

Comment:
as from product view non linux is the snowflake i don't see a reason for a linux group.

Reply:
currently "hosts: all" for 3 first tasks at dcos.yml, which makes extension to windows host groups as well. The idea was to split execution for hosts using ssh vs winrm.